settings.xml

```diff
@@ -0,0 +1,19 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
do we need this file here? remove it from repository

Reply:
done